src/hearings/containers/request-hearing/hearing-attendance/hearing-attendance.component.ts

```diff
@@ -88,7 +88,10 @@ export class HearingAttendanceComponent extends RequestHearingPageFlow implement
             partyType: partyDetail.partyType,
             partyRole: partyDetail.partyRole,
             partyName: `${partyDetail.individualDetails.firstName} ${partyDetail.individualDetails.lastName}`,
-            individualDetails: partyDetail.individualDetails,
+            individualDetails: {
+              ...partyDetail.individualDetails,
+              preferredHearingChannel: partyDetail.individualDetails?.preferredHearingChannel ? partyDetail.individualDetails?.preferredHearingChannel : ''
```

Comment:
Incredibly picky but if you are doing the optional chaining in the ternary test there's no point in doing it in the true path, but no need to change